### DISCUSSION TITLE
Welcome to Nudge-Post-install Discussions!

### DISCUSSION THREAD

**Discussion by dan-snelson** [dan-snelson/Nudge-Post-install#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

If you find Nudge-Post-install useful, please consider an upvote for [Suppport "Date" Key in Jamf Custom JSON Schema](https://www.jamf.com/jamf-nation/feature-requests/10232/).

**Comment by antcooley** [comment]
Hi! Thank you @dan-snelson for this clean and simple post install script for Nudge! I have one question for you: For the plist domain variable, where should that poing to? (Is this going to generate a plist file using the string for this variable or is it pointing to a location where a plist file is stored?) 
I'm new to scripting and I'm pretty comfortable with editing existing scripts but I just want to make sure I have everything set up correctly. 

Thanks again!

**Reply by dan-snelson** [reply]
Happy New Year, @antcooley!

You're correct: The [Reverse Domain Name](https://github.com/dan-snelson/Nudge-Post-install/wiki/4.-Script#jamf-pro-script-parameter-labels) will be used when [creating the various files client-side](https://github.com/dan-snelson/Nudge-Post-install/blob/main/Nudge-Post-install.bash#L67-L69).

**Reply by antcooley** [reply]
Awesome! Thank you!
